src/services/handlers/SendJobToLiltConnectorHandler.php

```diff
@@ -84,6 +84,8 @@ public function __invoke(Job $job): void
                     $draft
                 );
 
+                $slug = !empty($element->slug) ? $element->slug : '';
```

Comment:
what are the conditions where `$element->slug` is empty? maybe as the fallback we could have 'no-slug-available` just so it is clear even in the document name

Reply:
yes entries can be without slug, if it is just a data holders or so on. Yes I will add `no-slug-available`